### PR TITLE
manage-cluster-faq: add an faq to introduce how to estimate table size

### PR DESCRIPTION
**Pull request by TomShawn**
…of table

### What is changed, added or deleted? (Required)
manage-cluster-faq: add an faq to introduce how to estimate the size of a table.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: faq/manage-cluster-faq.md [hunk 1]
```diff
@@ -198,6 +198,40 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
...
+        AND table_name in (@table_name)
+    GROUP BY db_name , table_name , region_id) tabinfo
+GROUP BY db_name , table_name;
+```
+
+在使用以上语句时，你需要根据实际情况填写并替换掉语句里的以下字段：
+
+- `@dbname`：数据库名称
+- `@table_name`：目标表的名称
+- `@size_amplification`：集群压缩比的平均值。要获取该信息，查看 PD 监控下 **statistics balance** 面板下各节点的 `Size amplification` 指标，集群压缩比的平均值即为所有节点的 `Size amplification` 平均值。
```
Comment (TomShawn) [inline faq/manage-cluster-faq.md:231]: 这里是指 Grafana 监控面板吗？

Comment (Chad20N13) [reply faq/manage-cluster-faq.md:231]: 嗯是的

**Review by Chad20N13 — APPROVED** [review]
LGTM

**Review by Chad20N13 — APPROVED** [review]

File: faq/manage-cluster-faq.md (hunk 1 above)
Comment (TomShawn) [reply faq/manage-cluster-faq.md:231]:
```suggestion
- `@size_amplification`：集群压缩比的平均值。要获取该信息，查看 Grafana 监控下 **PD - statistics balance** 面板下各节点的 `Size amplification` 指标，集群压缩比的平均值即为所有节点的 `Size amplification` 平均值。
```

File: faq/manage-cluster-faq.md [hunk 2]
```diff
@@ -198,6 +198,40 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
...
+        AND table_name in (@table_name)
+    GROUP BY db_name , table_name , region_id) tabinfo
+GROUP BY db_name , table_name;
+```
+
+在使用以上语句时，你需要根据实际情况填写并替换掉语句里的以下字段：
+
+- `@dbname`：数据库名称
+- `@table_name`：目标表的名称
+- `@size_amplification`：集群压缩比的平均值。要获取该信息，查看 Grafana 监控下 **PD - statistics balance** 面板下各节点的 `Size amplification` 指标，集群压缩比的平均值即为所有节点的 `Size amplification` 平均值。
```
Comment (TomShawn) [inline faq/manage-cluster-faq.md:231]:
```suggestion
- `@size_amplification`：集群压缩比的平均值。要获取该信息，查看 Grafana 监控 **PD - statistics balance** 面板下各节点的 `Size amplification` 指标，集群压缩比的平均值即为所有节点的 `Size amplification` 平均值。
```

File: faq/manage-cluster-faq.md [hunk 3]
```diff
@@ -198,6 +198,40 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
 
 TiDB 在执行 SQL 时，预估出来每个 operator 处理了超过 10000 条数据就认为这条 query 是 expensive query。可以通过修改 tidb-server 配置参数来对这个门限值进行调整，调整后需要重新启动 tidb-server。
 
+### 如何预估 TiDB 中一张表的大小？
+
+要预估 TiDB 中一张表的大小，你可以参考使用以下查询语句：
+
```
Comment (TomShawn) [inline faq/manage-cluster-faq.md:204]:
```suggestion

{{< copyable "sql" >}}

```

File: faq/manage-cluster-faq.md [hunk 4]
```diff
@@ -198,6 +198,42 @@ information_schema 库中的 tables 表里的 create_time 即为表的真实创
...
+        db_name = @dbname
+        AND table_name in (@table_name)
+    GROUP BY db_name , table_name , region_id) tabinfo
+GROUP BY db_name , table_name;
+```
+
+在使用以上语句时，你需要根据实际情况填写并替换掉语句里的以下字段：
+
+- `@dbname`：数据库名称
+- `@table_name`：目标表的名称
```
Comment (lilin90) [inline faq/manage-cluster-faq.md:232]:
```suggestion
- `@dbname`：数据库名称。
- `@table_name`：目标表的名称。
```

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**